docs/crictl.md

```diff
@@ -0,0 +1,172 @@
+CRICTL User Guide
+=================
+This document presumes you already have `containerd` and `cri-containerd`
+installed and running.
+
+This document is for developers who wish to debug, inspect, and manage their pods,
+containers, and container images.
+
+Before generating issues against this document, `containerd`, `cri-containerd`,
+or `crictl` please make sure the issue has not already been submitted.
+
+## Install crictl
+If you have not already installed crictl please install the version compatible
+with the `cri-containerd` you are using. If you are a user, your deployement
```

Comment:
* If you are a user
* If you are a developer

Reply:
done